src/Sema.zig

```diff
@@ -21811,18 +21808,17 @@ fn reifyStruct(
         }
 
         if (any_aligned_fields) {
-            if (!try sema.intFitsInType(field_alignment_val, .u32, null)) {
-                return sema.fail(block, src, "alignment must fit in 'u32'", .{});
+            if (!try sema.intFitsInType(field_alignment_val, align_ty, null)) {
+                return sema.fail(block, src, "alignment must fit in '{f}'", .{align_ty.fmt(pt)});
             }
-
             const byte_align = try field_alignment_val.toUnsignedIntSema(pt);
             if (byte_align == 0) {
                 if (layout != .@"packed") {
-                    struct_type.field_aligns.get(ip)[field_idx] = .none;
+                    return sema.fail(block, src, "alignment can only be 0 on packed struct fields", .{});
```

Comment:
```suggestion
                    return sema.fail(block, src, "alignment must be >= 1", .{});
```
Don't tell me about the rules for reifying something *else*! Analagously, it'd be weird if `const S = struct { x: u32 align(0) };` told me about `packed struct`s. This also implicitly points users towards making their struct `packed` to solve the error, which isn't a good thing to be (unintentionally) suggesting.